app-quizz/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "bundler",
+    "moduleResolution": "node",
```

Comment:
Q: I think the correct option for module resolution in the bundler was selected.